app/javascript/dashboard/composables/useUISettings.js

```diff
@@ -114,6 +114,15 @@ const isEditorHotKeyEnabled = (key, uiSettings) => {
   return editorMessageKey === key;
 };
 
+/**
+ * Checks if RTL view is enabled in the UI settings.
+ * @param {Object} uiSettings - Reactive UI settings object.
+ * @returns {boolean} True if RTL view is enabled, otherwise false.
+ */
+const isRTLEnabled = uiSettings => {
+  return computed(() => uiSettings.value.rtl_view);
```

Comment:
@iamsivin Do we need this in the UI settings? Couldn't we just use the helper function directly in the components?

Reply:
@pranavrajs we don't need this function, it can be declared directly. I've made a commit to do this

Reply:
@scmmishra I don't think the change aligns with what I had in mind. Maybe I wasn't clear in the first comment. Let me try again.

I don't think this UI setting is required at all. We don't need to save it in the DB. It can be resolved dynamically from the selected language.

Instead of the following code.

```js
const { isRTL, updateUISettings } = useUISettings();
```

You could write the following. 

```js
const isRTL = getLanguageDirection(language);
```

Reply:
@pranavrajs Could please you take a look? I have updated the PR https://github.com/chatwoot/chatwoot/pull/9924/commits/f959f5218f059eb8528c5ebb3af9c0b24865f39d